metacat-main/src/main/java/com/netflix/metacat/main/services/impl/TableServiceImpl.java

```diff
@@ -109,6 +115,7 @@ public TableDto create(final QualifiedName name, final TableDto tableDto) {
         connectorTableServiceProxy.create(name, converterUtil.fromTableDto(tableDto));
```

Comment:
Should we run `saveParentChildRelationship(name, tableDto)` before `connectorTableServiceProxy.create`?

Pros:
1. To prevent read after write inconsistency, all the validations are done in mysql level, which are in `saveParentChildRelationship(name, tableDto)`. For now if the validation failed, we will need to drop the table as the table is already created. On the other hand, if we switch the order, if validation failed, we don't need to delete the table as it is not created.
2. For now, if there is any failure when saving parent child relation, we need to delete the created table. If we switch the order, if we fail to create the table, we just need to drop the parent child relation.